test/functional/wallet_balance.py

```diff
@@ -257,5 +259,22 @@ def test_balances(*, fee_node_1=0):
         assert_equal(self.nodes[0].getbalance(minconf=0), total_amount + 1)  # The reorg recovered our fee of 1 coin
 
 
+        def test_lastprocessedblock_json_field():
+            """
+            Tests the lastprocessedblock-object by comparing
+            the hashes & heights between generated blocks.
+            """
+            self.nodes[0].generatetoaddress(2, self.nodes[0].get_deterministic_priv_key().address)
```

Comment:
Please add test logging if you retouch this (see the logging in other tests).

Reply:
self.log.info is enough?

Reply:
(I edited my comment just above with a suggestion)